content/docs/1.4/scalers/cron.md

```diff
@@ -0,0 +1,51 @@
++++
```

Comment:
Can you move this to the v2.0 docs please? v1.4 is already shipped.

Reply:
@tomkerkhove do you want to have this as part of v2? If so, the code will need to be rebased, since now it is targeting v1.

Reply:
As discussed, this will stay with 1.0

Reply:
Agreed, but you'll need to create a 1.5 folder for all of the docs.

Reply:
@tomkerkhove we could probably prepare 1.5 docs in a separate PR?

Reply:
Definately, see https://github.com/kedacore/keda-docs/pull/177

Reply:
have moved the cron scaler to 1.5